docs/AboutUs.md

```diff
@@ -9,7 +9,7 @@ We are a team based in the [School of Computing, National University of Singapor
 
 ### Ang Jing Xuan Selwyn
 
-<img src="images/selwynang.png" width="200px">
+<img src="images/members/selwynang.png" width="200px">
```

Comment:
The filename instructed is `docs/images/github_username_in_lower_case.png` for scripts to detect our work, perhaps change them back?